Spanner/src/Session/CacheSessionPool.php

```diff
@@ -851,4 +852,123 @@ private function manageSessionsToDelete(array &$data)
             );
         }
     }
+
+    /**
+     * Maintain queued sessions for selected database and keep them alive.
+     *
+     * This method drops expired sessions and refreshes "old" ones (expiring in next 10 minutes).
+     * It can also refresh some non-"old" sessions to distribute refresh calls more or less
+     * evenly between maintenance calls.
+     * Only up to `minSessions` sessions are maintained, all excess ones are left to expire.
+     */
+    public function maintain()
+    {
+        if (!isset($this->database)) {
+            throw new \LogicException('Cannot maintain session pool: database not set.');
+        }
+
+        $this->config['lock']->synchronize(function () {
+            $cacheItem = $this->cacheItemPool->getItem($this->cacheKey);
+            $data = $cacheItem->get();
+            if (!$data) {
+                return;
+            }
+
+            $queue = $data['queue'];
```

Comment:
I understand that it's called a queue in existing code, but it'll make it more readable to just refer to them as `$sessions`, or even `$orderedSessions`. Or `$sessionsQueue`.

Reply:
Agree. `$sessions` will be better.